_examples/node-ts/webapp/client.gen.ts

```diff
@@ -123,10 +138,10 @@ const buildResponse = (res: Response): Promise<any> => {
     try {
       data = JSON.parse(text)
     } catch(err) {
-      throw { code: 'unknown', msg: `expecting JSON, got: ${text}`, status: res.status } as WebRPCError
+      throw new WebRPCError({ code: 'unknown', msg: `expecting JSON, got: ${text}`, status: res.status })
```

Comment:
I wonder if this shouldn't actually be wrapped in a WebRPCError. Instead, the try-catch could be removed, allowing the SyntaxError to bubble up for the caller to handle if they choose to. It should be a rare occurrence, and one that arguably should causes the whole application to blow up since it indicates something is very wrong with the webrpc server.